src/Zynga/Framework/PgData/V1/PgRow.hh

```diff
@@ -7,14 +7,19 @@ use Zynga\Framework\PgData\V1\Interfaces\PgRowInterface;
 use Zynga\Framework\PgData\V1\Exceptions\InvalidPrimaryKeyException;
 use Zynga\Framework\StorableObject\V1\Base as StorableObject;
 use Zynga\Framework\Type\V1\Interfaces\TypeInterface;
+use Zynga\Framework\Type\V1\BoolBox;
 
 abstract class PgRow extends StorableObject implements PgRowInterface {
   private PgModelInterface $_pgModel;
+  public BoolBox $_ignore_tombstoned;
 
   public function __construct(PgModelInterface $pgModel) {
 
     parent::__construct();
 
+    $this->_ignore_tombstoned = new BoolBox();
+    $this->_ignore_tombstoned->setIsRequired(false);
+
```

Comment:
I would add `$this->_ignore_tombstoned->set(false)` or `new BoolBox(false)`. The latter would need to add extra logic to the BoolBox constructer however. But by explicitly setting this field, it is explicitly clear what this box starting value it.